version.php

```diff
@@ -25,7 +25,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2018012400;
+$plugin->version   = 2018012401;
 $plugin->release   = 2018012400; // Match release exactly to version.
```

Comment:
The version for a plugin can just be the date (in this case), please keep the release in lock step too